ContactList/ContactList/UpdateController.swift

```diff
@@ -0,0 +1,391 @@
+                                 (Presentation(keyboardType: .numberPad, placeholder: phone, title: phone, dataType: .text(person.phoneNumber), cellType: .phoneNumber, validationType: .forPhoneNumber)),
+                                 (Presentation(keyboardType: .emailAddress, placeholder: email, title: email, dataType: .text(person.email), cellType: .email, validationType: .forEmail)),
+                                 (Presentation(placeholder: birthday, title: birthday, dataType: .date(person.birthday), cellType: .birthday)),
+                                 (Presentation(placeholder: height, title: height, dataType: .integer(person.height), cellType: .height)),
+                                 (Presentation(placeholder: note, title: note, dataType: .text(person.notes), cellType: .notes)),
+                                 (Presentation(title: driverLicense, dataType: .text(person.driverLicense), cellType: .driverLicenseSwitch))]
+        if !person.driverLicense.isEmpty {
+            array.append(getDriverLicenseNumberCell())
+        }
+        return array
+    }
+    
+    func getDriverLicenseNumberCell() -> Presentation {
+        let driverLicenseNumber = NSLocalizedString("DRIVER_LICENSE_NUMBER_LABLE_TITLE", comment: "Driver license number")
+        return Presentation(keyboardType: .default, placeholder: driverLicenseNumber, title: driverLicenseNumber, dataType: .text(currentPersonCopy.driverLicense), cellType: .driverLicense)
+    }
+    
+    func getCurentCell(at indexPath: IndexPath) -> UITableViewCell {
+        
+        switch indexPath.row {
+        case 0:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "ImageTableViewCell", for: indexPath) as! ImageTableViewCell
+            cell.setContent(cells[indexPath.row])
+            return cell
+        case 8:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "SwitchTableViewCell", for: indexPath) as! SwitchTableViewCell
+            cell.setContent(cells[indexPath.row])
+            cell.callback = {[weak self] (cell, isOn) in
+                let _ = self?.updatePersonInformation(index: indexPath.row, switchIsOn: isOn)
+            }
+            return cell
+        default:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "TextTableViewCell", for: indexPath) as! TextTableViewCell
+            cell.setContent(cells[indexPath.row])
+            cell.callback = {[weak self] (cell, text) in
+                return self?.updatePersonInformation(index: indexPath.row, text: text)
+            }
+            return cell
+        }
+    }
+    
+    func updatePersonInformation(index: Int, text: String = "", switchIsOn: Bool = false, image: UIImage? = nil) -> Bool?{
+        
+        var validationResult: Bool?
+        
+        if cells[index].cellType == .driverLicenseSwitch {
+            let indexPath = IndexPath(row: index + 1, section: 0)
```

Comment:
Again, try not to rely on hardcoded index, but try to calculate it dynamically
`let textfieldIndex = cells.firstIndex(where: { $0 == . driverLicense})`